ui/src/identity.ts

```diff
@@ -14,6 +14,8 @@ export interface Avatar {
 
 export interface Identity {
   id: string;
+  peerId: string;
+  urn: string;
```

Comment:
I'd be verrrry interested in seeing a `RadUrn` type at some point, but maybe not for now.

Reply:
Soon